source/index.html.erb

```diff
@@ -18,16 +18,14 @@ title: Index Page Example
     <div class="hero__content">
       <div class="grid-row">
         <div class="column-two-thirds">
-          <h1 class="hero__title">A hero unit</h1>
+          <h1 class="hero__title">Duis aute irure dolor in reprehenderit</h1>
           <p class="hero__description">
-            Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.
+            Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
           </p>
           <a href="#" role="button" class="hero-button">
-            Click on this call to action
-          </a>
-          <p class="hero__description">
-            Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur.
-          </p>
+            Do this thing
+          </a> <span class="hero-alternative-action"> or <a href="#">do another thing</a> because reasons</span>
+          <p>Pellentesque commodo arcu in sollicitudin lacinia. Vivamus lacus nibh, maximus nec laoreet eget, condimentum vel mi.</p>
```

Comment:
If there's text after the button, it's not in the same font size as the description, it behaves like a normal paragraph.